### PR TITLE
Add recent activity feed to admin dashboard

### LINKED ISSUE #489: I would like to see a recent activity feed
Fred Jones signed up for Something Something (time / date)
“this is great!” said [Judy Smith](link/to/profile) on [Kids.gov Twitter](link/to/task)  (time / date)
Jody Tuan posted a new opportunity: "Stuff Should Happen!"
Tom Smith updated his profile (time / date)
Betty Rubble joined! (time / date)

### PR DESCRIPTION
**Pull request by dhcole**
Shows admins recent activities of all users. Should replace the need to have admins get all email notifications.

Discussion in #489
Closes #489
Refs #487

### CODE REVIEW

File: api/controllers/AdminController.js [hunk 1]
```diff
@@ -279,6 +279,202 @@ module.exports = {
...
+    var actions = {
+      projectCommentAdded: templates.newComment,
+      taskCommentAdded: templates.newComment,
+      taskVolunteerAdded: templates.newVolunteer,
+      welcomeUser: templates.newUser,
+      taskCreated: templates.newTask
+    };
+
+    // Get active notifications
+    Notification.find({ isActive: true }).sort('createdAt desc').exec(next);
```
Comment (polastre) [inline api/controllers/AdminController.js]: Should probably include a `limit` clause on this so you only get the results that you need from the DB.

**Comment by polastre** [comment]
We should standardize on how we pass error messages/objects.  In some of our code, we pass an object like: 
```js
{
  err: 'foo'
  message: 'bar'
}
```

In other places (like here), we're passing strings as the error object.

This isn't a big deal, but we should standardize (and log!) the errors going forward.

**Comment by dhcole** [comment]
@polastre  8359b19 adds pagination. Will standard on that error format going forward.

@ultrasaurus this is good to go from my end.

**Comment by polastre** [comment]
Looks good to me, although I'm not a fan of commented out code like in 83b5198

**Comment by dhcole** [comment]
I removed the commented code. This should be good to go.

**Pull request merged**